generators/cucumber/templates/env.rb

```diff
@@ -2,38 +2,40 @@
 require 'capybara/cucumber'
 require_relative '../../boot'
 
-World(Capybara::Settings)
+World(Howitzer::CapybaraSettings)
 World(FactoryGirl::Syntax::Methods)
 
 log.settings_as_formatted_text
-DataStorage.store('sauce', :start_time, Time.now.utc)
-DataStorage.store('sauce', :status, true)
+Howitzer::Utils::DataStorage.store('sauce', :start_time, Time.now.utc)
```

Comment:
It should be tested on real project in elegant way. I put TODO at the end of file. We will improve that during howitzer_examples migration